src/browser/app/profile/better-fox.js

```diff
@@ -1,4 +1,4 @@
-// Ignore license in this file
```

Comment:
You need to keep this comment at the stop of this file otherwise gluon will complain about it not having MPL license headers

Reply:
Right, sorry

Reply:
This is fixed